x/hard/keeper/liquidation.go

```diff
@@ -187,6 +187,10 @@ func (k Keeper) StartAuctions(ctx sdk.Context, borrower sdk.AccAddress, borrows,
 				if err != nil {
 					return liquidatedCoins, err
 				}
+				// Decrement supplied coins and increment borrowed coins optimistically
+				k.DecrementSuppliedCoins(ctx, sdk.Coins{lot})
+				k.IncrementBorrowedCoins(ctx, sdk.Coins{bid})
```

Comment:
Total borrow should be decremented.
On a succesful auction, the borrow amount is returned to the module account, cancelling out the debt. So total borrow should decrease.